src/main/scala/com/vngrs/etl/Loader.scala

```diff
@@ -0,0 +1,31 @@
+package com.vngrs.etl
+
+import org.apache.spark.rdd.RDD
+import org.apache.spark.sql.Row
+
+trait Loader[A] {
+
+  protected[etl] def apply(input: RDD[A]): Loader[A]
```

Comment:
I was kind of undecided on that. I was thinking

```scala
val pipeline: Pipeline = //.......

pipeline
  .load(Loader.file())
  .load(Loader.redshift())
```

could be cool. What do you think about it?